src/pages/[platform]/build-a-backend/data/customize-authz/index.mdx

```diff
@@ -144,6 +144,44 @@ const schema = a.schema({
 });
 ```
 
+### Non-model authorization rules
+
+**Non-model** types are any types added to the schema without using `a.model()`. These consist of modifiers such as `a.customType()`, `a.enum()`,`a.query()`, `a.mutation()`, or `a.subscription()`.
+
+Dynamic authorization rules such as `allow.owner()`, `allow.ownerDefinedIn()`, `allow.groupDefinedIn()` are not supported for **non-model** types.
+
+**Example**
+
+```ts
+// Custom Query
+listCustomType: a
```

Comment:
does the syntax highlighting render okay? previous examples wrap with `a.schema()`

Reply:
Yeah, looks okay on render:

![CleanShot 2024-07-12 at 12 52 40@2x](https://github.com/user-attachments/assets/e5d5757f-c496-4ee9-95d8-39e4cf3eb275)

Reply:
can we wrap it with `a.schema()` for consistency with the other examples?

Reply:
Sure! pushed new commit